components/document-link/index.jsx

```diff
@@ -1,22 +1,21 @@
 import React from 'react'
 
-import { Alert, Icon } from 'design'
+import styles from './styles.css'
+
+import { Button, Icon } from 'design'
 
 const DocumentLink = React.memo(({ href, children }) => (
-  <a
+  <Button
+    tag="a"
     href={href}
-    // We want to pass referrer tag to our website
-    // eslint-disable-next-line react/jsx-no-target-blank
```

Comment:
Oh, interesting issue. I see we have to add `Button` to the list of components that the rule has to check. This comment should stay.

Reply:
is it possible to configure it?

Reply:
Check the [custom link components option](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md#custom-link-components).

Also, we may want to enable `allow-referrer` option.

Reply:
That's quite clever 👍